gh-badges/templates/default-template.json

```diff
@@ -5,5 +5,6 @@
   "message": {{=JSON.stringify(it.text[1])}},{{?it.logoWidth}}
   "logoWidth": {{=JSON.stringify(it.logoWidth)}},{{?}}{{?it.labelColor}}
   "labelColor": {{=JSON.stringify(it.labelColor)}},{{?}}
-  "color": {{=JSON.stringify(it.color || null)}}
+  "color": {{=JSON.stringify(it.color || null)}}{{?(it.links[0] && it.links[0].length || it.links[1] && it.links[1].length)}},
```

Comment:
Should the comma on this line be inbetween the 2 template expressions, like it is on the previous lines?

Reply:
`color` may be the final element in the JSON payload if there isn't a `link` element, therefore the comma has to be part of the last template expression. 😉